config/rbac/role.yaml

```diff
@@ -24,6 +24,8 @@ rules:
   - persistentvolumeclaims
   - persistentvolumes
   verbs:
+  - create
+  - delete
```

Comment:
@coreydaley does the operator delete any pvc/pv ? I guess it would be safer if we would not allow deletion of pvc, right?

Reply:
Testing this pull request did give me an error when it did not have the `delete` permission, so if we want to exclude it from ever being deleted that probably needs to be built into the code somewhere.